jac-uk/admin
# #1380 - User Roles Phase 1

## What's included?
The ability to create roles, set permissions and assign roles to users.

The first phase of User Roles is only to set up roles and users - the second phase will be to implement the restrictions everywhere in the system.

## Who should test?
✅ Product owner
✅ Developers
✅ UTG

## How to test?
Log in as a digital user, click users. 

Disable / enable users and try to log in.

Create roles with different permissions! Most of it should be self-explanatory.

## Risk - how likely is this to impact other areas?
🔴 High risk - this includes a lot of changes to shared code

---
PREVIEW:DEVELOP
_can be OFF, DEVELOP or STAGING_
